multilep/interface/GenTools.h

```diff
@@ -31,14 +31,17 @@ namespace GenTools{
     bool photonInChain(const std::set<int>&);
     bool udsInChain(const std::set<int>&);
     bool tauInChain(const std::set<int>&);
+    bool heavyNeutrinoInChain(const std::set<int>&);
     //find the provenance of a particle using the contents of its decayChain
     unsigned provenance(const reco::GenParticle*, const std::vector<reco::GenParticle>&);
     unsigned provenanceCompressed(const reco::GenParticle*, const std::vector<reco::GenParticle>&, bool isPrompt);
 
     //check whether photon comes from ME in conversion
     unsigned provenanceConversion(const reco::GenParticle*, const std::vector<reco::GenParticle>&);
+    unsigned provenanceHeavyNeutrino(const reco::GenParticle&, const std::vector<reco::GenParticle>&);
```

Comment:
This should either be int, or the corresponding branch should be unsigned.

Reply:
The corresponding branch is now unsigned